### PR TITLE
ios 14 data collection

### PR DESCRIPTION
**Pull request by davidbielik**
* apple developer questionnaire description

### CODE REVIEW

**Review by KellieHawks — APPROVED** [review]
lgtm! Let me know if you'd like me to merge without Jared's review.

File: _docs/_developer_guide/platform_integration_guides/ios/ios_14.md [hunk 1]
```diff
@@ -107,9 +107,35 @@ No changes to Provisional Push Authorization are included in iOS 14. In an earli
 
 ### App Privacy and Data Collection Overview {#app-privacy}
 
-The App Store's new App Privacy feature will disclose to users what personal information an app collects and how it may track users across other apps and websites. Apple has [stated](https://www.apple.com/ios/ios-14-preview/), "Privacy information on the App Store will be coming in an iOS 14 update later this year".
+Starting on Dec 8, 2020, all submissions to the App Store require additional steps to adhere to [Apple's new App Privacy standards](https://developer.apple.com/app-store/app-privacy-details/).
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/ios_14.md:110]: Starting on -> Since

**Review by Bucimis — APPROVED** [review]
thanks for writing this up

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Pull request merged**